### DISCUSSION TITLE
Integrate Komodo wallet

### DISCUSSION THREAD

**Discussion by biishnuthapa** [rainbow-me/rainbowkit#1802] [Ideas]
Can you please integrate komodo wallet also

**Comment by 0xmkh** [comment]
@biishnuthapa Does komodo wallet support evm chains ?

**Comment by biishnuthapa** [comment]
No komodo wallet doesn't support evm chains??

**Reply by 0xmkh** [reply]
We would love to have a wallet that supports evm chains. Do you have any other options ?
